src/groovy/org/pih/warehouse/inventory/StockMovementStatusCode.groovy

```diff
@@ -27,6 +27,7 @@ enum StockMovementStatusCode {
     PACKED(7, PENDING),
     REVIEWING(8, PENDING),
     DISPATCHED(9),
+    ISSUED(9),
```

Comment:
Let's just use DISPATCHED instead of adding this. I know it's a bit confusing but if we're going to change this back to ISSUED I'd like to do that as a separate refactoring.